workers/unity/Assets/Playground/Scripts/Player/ConnectButtonClickSystem.cs

```diff
@@ -0,0 +1,99 @@
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.UI;
+#if UNITY_ANDROID
+using Improbable.Gdk.Android;
+
+#endif
+
+namespace Playground
+{
+    [UpdateBefore(typeof(ConnectClientSystem))]
+    [UpdateBefore(typeof(SpatialOSUpdateGroup))]
```

Comment:
I'm not sure if that works. I believe you can only have one type of tag at a time